src/Chip/index.scss

```diff
@@ -0,0 +1,71 @@
+%chipAlignedContent {
+  height: rem(24px);
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nds-chip {
+  @extend %chipAlignedContent;
+  border-radius: rem(24px);
+  font-size: var(--font-size-s);
+  padding: rem(2px) var(--space-s);
+  width: fit-content;
+  max-width: 60ch; // label truncates when max constraint is reached.
```

Comment:
It's 60 chars width for the entire chip - so more like 40-50 for the label, depending on icons and such